sha/host/src/main.rs

```diff
@@ -0,0 +1,27 @@
+use methods::{HASH_ID, HASH_PATH};
+use risc0_zkp::core::sha::Digest;
+use risc0_zkvm::host::Prover;
+use risc0_zkvm::serde::{from_slice, to_vec};
+
+fn main() {
+    // Make the prover.
+    let method_code =
+        std::fs::read(HASH_PATH).expect("Method code should be present at the specified path");
```

Comment:
Prefer `HASH_CONTENTS` so that you don't need disk I/O.

Reply:
Hmm, is there a trick to this? Trying
```
use methods::{HASH_CONTENTS, HASH_ID, HASH_PATH};
```
gives the error
```
no `HASH_CONTENTS` in the root
```

Reply:
Ah, this is probably only available on 0.12, sorry :)